### PR TITLE
Rework config setup to better separate dev and prod

### PR DESCRIPTION
**Pull request by lmorchard**
Issue #97

### CODE REVIEW

**Comment by lmorchard** [comment]
I'm not 100% sure this is all correct - seems to still work for local & dev deployments, but probably will need some further iteration to be right for prod/stage deployments

**Comment by lmorchard** [comment]
@oremj Any thoughts on what needs to be tweaked here to move forward on deployment stuff?

File: config/production.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+custom:
+  customDomain:
+    enabled: false
+
+  resourceNames:
+    hitrate: ${env:HITRATE_TABLE}
+    credentials: ${env:CREDENTIALS_TABLE}
+    sqs: ${env:QUEUE_NAME}
+    sqsArn: ${env.QUEUE_ARN}
```
Comment (chenba) [inline config/production.yml]: `${env:QUEUE_ARN}` (colon instead of period)?

Comment (lmorchard) [reply config/production.yml]: Oh, yup! Good catch!

**Comment by lmorchard** [comment]
Gave this a rebase against master after merging the email thing. Going to give it another try or two on the dev stack before merging. Kind of expecting some further iteration when rubber meets road in a real stage / prod deployment attempt.

**Pull request merged**